app/components/header/header.component.html

```diff
@@ -465,6 +465,26 @@ <h5 type="text">Upstream:</h5>
   </div>
 </div>
 
+<!--Modal for commiting from navbar-->
+<div id="set-commit-modal" class="modal fade" tabindex="-1" role="dialog" aria-labelledby="mySmallModalLabel"
+  aria-hidden="true">
+  <div class="modal-dialog modal-sm">
+    <div class="modal-content" style=" width: 602px !important">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-hidden="true">&times;</button>
+        <h4 class="modal-title">Commit changes</h4>
+      </div>
+      <div class="modal-body" id="modal-text-box">
+        Please provide a commit message.
+        <input type="text" id="commit-message-input-navbar" class="form-control" style=" width: 554px !important" placeholder="Enter a commit message">
```

Comment:
I am wondering if we should pre-populate this textbox with the text entered for a commit message on the main screen?